distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -1090,3 +1056,40 @@ Text.write path encoding=Encoding.utf_8 on_existing_file=Existing_File_Behavior.
     file = new path
     on_existing_file.write file stream->
         stream.write_bytes bytes
+
+## Writes (or appends) the Vector of bytes into the specified file. The behavior
+   specified in the `existing_file` parameter will be used if the file exists.
+
+   Arguments:
+   - path: The path to the target file.
+   - on_existing_file: Specifies how to proceed if the file already exists.
+
+   If the Vector contains any item which is not a `Byte`, an
+   `Illegal_Argument_Error` will be raised. Enso follows the Java convention,
+   that a `Byte` is between -128 and 127.
+   If the path to the parent location cannot be found or the filename is
+   invalid, a `File_Not_Found` is raised.
+   If another error occurs, such as access denied, an `Io_Error` is raised.
+   Otherwise, the file is created with the encoded text written to it.
+
+   > Example
+     Write the bytes of the text "hello" to a file.
+
+         import Standard.Examples
+
+         example_write_bytes = [32, 63, 95, -120, -43].write_bytes Examples.scratch_file
```

Comment:
Is the comment really true? I'm pretty sure `32` is does _not_ stand for `h` in ASCII.